modules/cdp-engine/src/lib.rs

```diff
@@ -112,8 +112,9 @@ type ChangeBalance = Change<Balance>;
 /// Liquidation strategy available
 #[derive(Encode, Decode, Clone, RuntimeDebug, PartialEq, Eq, TypeInfo)]
 pub enum LiquidationStrategy {
-	/// Liquidation CDP's collateral by create collateral auction
-	Auction,
+	/// Liquidation CDP's collateral by create collateral auction, u32 represents number of auctions
+	/// created
+	Auction(u32),
```

Comment:
maybe this is more self-documenting?
```suggestion
	Auction { count: u32 },
```